pkgs/development/python-modules/weasyprint/default.nix

```diff
@@ -43,9 +43,15 @@ buildPythonPackage rec {
 
   FONTCONFIG_FILE = "${fontconfig.out}/etc/fonts/fonts.conf";
 
-  propagatedBuildInputs = [ cairosvg pyphen cffi cssselect lxml html5lib tinycss pygobject2 ];
+  propagatedBuildInputs = [ cairosvg pyphen cffi cssselect lxml html5lib tinycss ];
 
+  # pango patch can be removed after next release of weasyprint, see:
+  # https://github.com/Kozea/WeasyPrint/issues/1333#issuecomment-818062970
```

Comment:
Bonus points: Move this just above `fetchpatch`, as that makes it clearer, that this is the "pango patch".

Reply:
In other reviews I'd been told to keep comments outside of the function(?). Changed `pango` to just the name of the patch.

Reply:
There is no harm in putting the comment closer.

Reply:
Good to know. That said since I've already made a different change, is it now sufficient?

Reply:
Yeah, I'm not going to nitpick, just testing the build rn.